src/Serilog.Sinks.ElmahIO/LoggerConfigurationElmahIOExtensions.cs

```diff
@@ -36,7 +36,7 @@ public static class LoggerConfigurationElmahIOExtensions
         public static LoggerConfiguration ElmahIO(
             this LoggerSinkConfiguration loggerConfiguration,
              Guid logId,
-            LogEventLevel restrictedToMinimumLevel = LogEventLevel.Error,
+            LogEventLevel restrictedToMinimumLevel = LevelAlias.Minimum,
```

Comment:
We are working on the new Elasticsearch sink and started to document stuff in the wiki over there. Looks like that is a nice way forward. 
I m not an admin in this site, but it looks like I can create Wiki pages. Can you also do that? If not, then I can create some details.